### PR TITLE
[WIP] Docs/networks/lukso tool.md

### PR DESCRIPTION
**Pull request by patred20**

### CODE REVIEW

**Review by frozeman — APPROVED** [review]

**Comment by mxmar** [comment]
@patred20 Is this still `WIP` (`Draft`) PR?

**Review by mxmar — CHANGES_REQUESTED** [review]
Missing information with example how exactly run `lukso` binary after installer finished its work.

1. Description step-by-step
2. Example

File: docs/networks/lukso-tool.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Lukso setup/cli tool
```
Comment (frozeman) [inline docs/networks/lukso-tool.md:1]: LUKSO

Comment (frozeman) [reply docs/networks/lukso-tool.md:1]: LUKSO CLI

**Comment by frozeman** [comment]
lukso-cli.md

File: docs/networks/lukso-tool.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Installation
+`curl https://install.l15.lukso.network | bash`
+This shell script will: 
+1. Create directory under `/opt/lukso`
+2. Download zipped files required for node startup.
+3. Unzip them (using `unzip` tool)
+4. Create symbolic link in `/usr/local/bin`.
+
+## Config file
```
Comment (frozeman) [inline docs/networks/lukso-tool.md:13]: `## Example config file`

**Review by frozeman — APPROVED** [review]

**Review by frozeman — APPROVED** [review]
good

**Pull request merged**